babel.config.json

```diff
@@ -5,11 +5,7 @@
       "bugfixes": true,
       "shippedProposals": true,
       "corejs": 3,
-      "useBuiltIns": "entry",
-      "include": [
-        "@babel/proposal-nullish-coalescing-operator",
-        "@babel/proposal-optional-chaining"
```

Comment:
These are included anyway as long as we support "maintained node versions"
https://github.com/adazzle/react-data-grid/blob/canary/browserslist#L5